src/Compound/CompoundGenerator.php

```diff
@@ -1,11 +1,14 @@
 <?php
-namespace Hostnet\Component\EntityPlugin;
+namespace Hostnet\Component\EntityPlugin\Compound;
 
 use Composer\IO\IOInterface;
 use Composer\Package\PackageInterface;
+use Hostnet\Component\EntityPlugin\EntityPackage;
+use Hostnet\Component\EntityPlugin\PackageClass;
+use Hostnet\Component\EntityPlugin\WriterInterface;
 
 /**
- * The generator for stage 2 that only has to hook into composer
+ * The generator for stage 1 that only has to hook into composer
```

Comment:
what is stage 1? Maybe a better description without needing to have knowledge of the internals of composer here.